src/components/navigation/Navbar.tsx

```diff
@@ -185,9 +208,21 @@ const Navbar = ({ children, logo, linkComponent, ...props }: NavbarProps) => {
               </Box>
             </Link>
           )}
+          {logoRightSlot}
           {children}
```

Comment:
Maybe it makes more sense to add the children by default into the logoRightSlot, rather than a separate "special" slot. 

(Then maybe it should be called "leftSlot" to mirror "rightSlot...)

Reply:
Removed logoLeftSlot and now leftSlot content gets children too.